code/modules/mob/living/carbon/superior_animal/roach/types/kaiser.dm

```diff
@@ -0,0 +1,131 @@
+	var/health_marker_2 = 1000
+	var/health_marker_3 = 500
+
+	blattedin_revives_left = 0
+
+	// TODO: Add a special type of meat for Kaiser
+	meat_type = /obj/item/weapon/reagent_containers/food/snacks/meat/roachmeat/fuhrer
+	meat_amount = 15
+	sanity_damage = 3
+
+/mob/living/carbon/superior_animal/roach/kaiser/New()
+	..()
+	gas_sac = new /datum/reagents(100, src)
+	pixel_x = -16  // For some reason it doesn't work when I overload them in class definition, so here it is.
+	pixel_y = -16
+
+
+/mob/living/carbon/superior_animal/roach/kaiser/Life()
+	. = ..()
+	if(stat != CONSCIOUS)
+		return
+
+	if(stat != AI_inactive)
+		return
+	
+	if(can_call_reinforcements())
+		distress_call()
+
+	gas_sac.add_reagent("blattedin", 1)
+	if(prob(7))
+		gas_attack()
+
+
+// TOXIC ABILITIES
+/mob/living/carbon/superior_animal/roach/kaiser/UnarmedAttack(atom/A, proximity)
+	. = ..()
+
+	if(isliving(A))
+		var/mob/living/L = A
+		if(istype(L) && prob(10))
+			var/damage = rand(melee_damage_lower, melee_damage_upper)
+			L.adjustToxLoss(damage)
+			playsound(src, 'sound/voice/insect_battle_screeching.ogg', 30, 1, -3)
+			L.visible_message(SPAN_DANGER("\the [src] globs up some toxic bile all over \the [L]!"))
+
+// SUPPORT ABILITIES
+/mob/living/carbon/superior_animal/roach/kaiser/proc/gas_attack()
+	if (!gas_sac.has_reagent("blattedin", 20) || stat != CONSCIOUS)
+		return
+
+	var/location = get_turf(src)
+	var/datum/effect/effect/system/smoke_spread/chem/S = new
+
+	S.attach(location)
+	S.set_up(gas_sac, gas_sac.total_volume, 0, location)
+	src.visible_message(SPAN_DANGER("\the [src] secretes strange vapors!"))
+
+	spawn(0)
+		S.start()
+
+	gas_sac.clear_reagents()
+	return TRUE
+
+/mob/living/carbon/superior_animal/roach/support/findTarget()
+	. = ..()
+	if(. && gas_attack())
+		visible_emote("charges at [.] in clouds of poison!")
+
+// FUHRER ABILITIES
+/mob/living/carbon/superior_animal/roach/kaiser/proc/distress_call()
+	if (!distress_call_stage)
+		return
+
+	for (var/mob/living/carbon/human/H in view())
+		if (H.stat != DEAD && H.client)
+			break
+
+	if (distress_call_stage)
+		distress_call_stage--
+		playsound(src.loc, 'sound/voice/shriek1.ogg', 100, 1, 8, 8)
+		spawn(2)
+			playsound(src.loc, 'sound/voice/shriek1.ogg', 100, 1, 8, 8)
+		visible_message(SPAN_DANGER("[src] emits a horrifying wail as nearby burrows stir to life!"))
+		for (var/obj/structure/burrow/B in find_nearby_burrows(src))
+			B.distress(TRUE)
+
+
+/mob/living/carbon/superior_animal/roach/kaiser/proc/can_call_reinforcements()
+	if(health_marker_1 >= health && health > health_marker_2 && distress_call_stage == 3)
+		return TRUE
+	if(health_marker_2 >= health && health > health_marker_3 && distress_call_stage == 2)
+		return TRUE
+	if(health_marker_3 >= health && health > 0 && distress_call_stage == 1)
+		return TRUE
+	return FALSE
+
+/mob/living/carbon/superior_animal/roach/kaiser/slip(var/slipped_on)
+	return FALSE
+
+// Kaiser has no death sprite, so he explodes when dies. 
```

Comment:
A little disappointing, should drop a trophy item for bragging rights, like a kaiser roach head or something

Reply:
I have no Kaiser head sprite. It won't be too difficult to add it when someone will draw it.

Reply:
This can be done in future PRs.